src/libs/actions/Report.ts

```diff
@@ -738,7 +738,7 @@ function openReport(
     if (ReportUtils.isGroupChat(newReportObject)) {
         parameters.chatType = CONST.REPORT.CHAT_TYPE.GROUP;
         parameters.groupChatAdminLogins = currentUserEmail;
-        parameters.optimisticAccountIDList = participantAccountIDList.join(',');
+        parameters.optimisticAccountIDList = Object.keys(newReportObject.participants ?? []).join(',');
```

Comment:
The `?? []` needs to be `?? {}` ? It's an object not an array.

Reply:
Nice catch. The good news is that JS doesn't seem to care haha
![Screenshot 2024-04-25 at 4 49 02 PM](https://github.com/Expensify/App/assets/22219519/4ae6ffbb-6716-4f04-ba59-d952a7d11a73)

Reply:
Nice, that's better than returning a `0`. 😄  Would be great to fix it though.

Reply:
Agree

Reply:
Oh shit, Carlos test was exactly what I did in the console, creating a PR to fix this.

Reply:
https://github.com/Expensify/App/pull/41043